src/rules/speech_control.ts

```diff
@@ -1178,4 +1178,36 @@ export function initRules_bc_speech_control() {
 			}, ModuleCategory.Rules);
 		}
 	});
+
+	registerRule("speech_garble_while_talking", {
+		name: "Force garbled speech",
+		type: RuleType.Speech,
+		loggable: false,
+		shortDescription: "force PLAYER_NAME to talk as if they were gagged",
+		longDescription: "This rule forces PLAYER_NAME to talk as if they were gagged, automatically garbling all of their speech. This rule does not affect OOC or, if \"Garble whispers while gagged\" is disabled, whispers.",
+		keywords: ["garble", "saying", "talking"],
+		defaultLimit: ConditionsLimit.normal,
+		dataDefinition: {
+			gagLevel: {
+				type: "number",
+				default: 5,
+				options: {
+					min: 1,
+					max: 25
```

Comment:
This feels like arbitrary constant, why `25`? As far as I can see the highest level that changes something is 20.
Also alternative could be using `SpeechGagLevelLookup.GagTotal4` instead of numeric constant.

Reply:
Correct, this number in particular was chosen more so with FBC in mind whose "Extra gag anti-cheat" option adds additional extra-heavy garbling at gag levels of >24, hence why I put it at 25.

Reply:
Hm, on second thought: if you're okay with leaving it at 25 it might be good to attach a comment with the rationale though.

Reply:
Yes, if FBC adds those, then it would be nice to leave it as-is, but add a comment as to why it was chosen.